card/mod/03_machines/set/self/script_slot.rb

```diff
@@ -1,5 +1,7 @@
 include_set Abstract::CodeFile
 
 def source_files
-  %w( wagn_mod.js.coffee wagn.js.coffee )
+  %w( wagn_mod wagn_editor wagn_layout wagn_navbox wagn_upload wagn ).map do |n|
+    "#{n}.js.coffee"
+  end << 'script_prosemirror.js'
```

Comment:
I'm not sure if we want prosemirror in here or add it with a migration as separate card to *all+*script. We have a separate card for ace editor code but wagn_editor.js.coffee doesn't work without the ace and prosemirror library.